examples/peripherals/timer_group/main/timer_group_example_main.c

```diff
@@ -70,7 +44,7 @@ void IRAM_ATTR timer_group0_isr(void *para)
     /* Prepare basic event data
        that will be then sent back to the main program task */
     timer_event_t evt;
-    evt.timer_group = 0;
+    evt.timer_group = timer_group_t(0);
```

Comment:
This isn't idiomatic C, and it's not strictly necessary in C either. Suggest leaving it off.

Reply:
If you needed this so this code worked as C++, you can use `(timer_group_t) 0` which is idiomatic C and accepted by a C++ compiler.

Reply:
Actually, on reflection using the enum value `TIMER_GROUP_0` is probably the best choice here.